spec/vcr/applications_controller/show.yml

```diff
@@ -99,7 +99,7 @@ http_interactions:
       - chunked
     body:
       encoding: UTF-8
-      string: '{"totalSize":1,"done":true,"records":[{"attributes":{"type":"Application__c","url":"/services/data/v43.0/sobjects/Application__c/a0o0U000000VVw8QAG"},"Id":"a0o0U000000VVw8QAG","Name":"APP-00262254","Applicant__c":"a0n0U0000030tBrQAI","Alternate_Contact__c":null,"Listing__c":"a0W0U000000MX4vUAG","Listing__r":{"attributes":{"type":"Listing__c","url":"/services/data/v43.0/sobjects/Listing__c/a0W0U000000MX4vUAG"},"Name":"LAP
+      string: '{"totalSize":1,"done":true,"records":[{"attributes":{"type":"Application__c","url":"/services/data/v43.0/sobjects/Application__c/a0o0U000000VVw8QAG"},"Id":"a0o0U000000VVw8QAG","Name":"APP-00262254","Applicant__c":"a0n0U0000030tBrQAI","Primary_Applicant__c":"0030U00000P8lxaQAB","Alternate_Contact__c":null,"Listing__c":"a0W0U000000MX4vUAG","Listing__r":{"attributes":{"type":"Listing__c","url":"/services/data/v43.0/sobjects/Listing__c/a0W0U000000MX4vUAG"},"Name":"LAP
```

Comment:
Primary_applicant__c is the only piece that's new potential user info and it's definitely fake. "Lappy Laptest" https://sfhousing--full.cs97.my.salesforce.com/0030U00000P8lxaQAB